configure.sh

```diff
@@ -102,4 +104,95 @@ else
     -DLWS_WITHOUT_DAEMONIZE=ON -DCMAKE_C_FLAGS=-fPIC -DLWS_WITH_NO_LOGS=ON -DCMAKE_BUILD_TYPE=Release
     make $@
 fi
-popd > /dev/null # ${FRAMEWORK_DIR}
\ No newline at end of file
+popd > /dev/null
+
+pushd ${FRAMEWORK_DIR} > /dev/null
+if [ "$TARGET" = "arm" ]; then
+    TARGET=arm
+else
+    TARGET=linux
+fi
+
+if [ "$TARGET" = "arm" ]; then
+    search_paths=$SDKTARGETSYSROOT
```

Comment:
not required under arm because there will never be SDKTARGETSYSROOT , you always have to build.